sshd.c

```diff
@@ -884,16 +897,16 @@ privsep_postauth(Authctxt *authctxt)
 		    posix_spawn_file_actions_adddup2(&actions, tmp_sock, STDIN_FILENO) != 0 ||
 		    posix_spawn_file_actions_adddup2(&actions, tmp_sock, STDOUT_FILENO) != 0 ||
```

Comment:
why not STDERR_FILENO?

Reply:
See fork logic for Unix. You don't touch stderr - i.e you let child inherit parent's stderr.